### PR TITLE
Docs: Document caching defaults in enterprise config

### PR DESCRIPTION
**Pull request by hairyhenderson**
**What this PR does / why we need it**:

Documents the default caching values in the enterprise-configuration docs.

**Which issue(s) this PR fixes**:

n/a (result of internal discussion)

**Special notes for your reviewer**:

Signed-off-by: Dave Henderson <dave.henderson@grafana.com>

### CODE REVIEW

File: docs/sources/enterprise/enterprise-configuration.md [hunk 1]
```diff
@@ -357,33 +359,44 @@ _Time to live_ (TTL) is the time that a query result is stored in the caching sy
...
 
+The default is `1`.
+
 ## [caching.memory]
 
 ### gc_interval
 
 When storing cache data in-memory, this setting defines how often a background process cleans up stale data from the in-memory cache. More frequent "garbage collection" can keep memory usage from climbing but will increase CPU usage.
 
+The default is `1m`
```
Comment (oddlittlebird) [inline docs/sources/enterprise/enterprise-configuration.md:370]:
```suggestion
The default is `1m`.
```

File: docs/sources/enterprise/enterprise-configuration.md [hunk 2]
```diff
@@ -357,33 +359,44 @@ _Time to live_ (TTL) is the time that a query result is stored in the caching sy
...
 
+The default is `"grafana"`.
+
 ## [caching.memcached]
 
 ### memcached_servers
 
 A space-separated list of memcached servers. Example: `memcached-server-1:11211 memcached-server-2:11212 memcached-server-3:11211`. Or if there's only one server: `memcached-server:11211`
+
+The default is `"localhost:11211"`
```
Comment (oddlittlebird) [inline docs/sources/enterprise/enterprise-configuration.md:402]:
```suggestion
The default is `"localhost:11211"`.
```

File: docs/sources/enterprise/enterprise-configuration.md [hunk 3]
```diff
@@ -357,33 +359,44 @@ _Time to live_ (TTL) is the time that a query result is stored in the caching sy
...
 
 A string that prefixes all Redis keys. This value must be set if using a shared database in Redis. If `prefix` is empty, then one will not be used.
 
+The default is `"grafana"`.
+
 ## [caching.memcached]
 
 ### memcached_servers
 
 A space-separated list of memcached servers. Example: `memcached-server-1:11211 memcached-server-2:11212 memcached-server-3:11211`. Or if there's only one server: `memcached-server:11211`
```
Comment (oddlittlebird) [inline docs/sources/enterprise/enterprise-configuration.md:400]:
```suggestion
A space-separated list of memcached servers. Example: `memcached-server-1:11211 memcached-server-2:11212 memcached-server-3:11211`. Or if there's only one server: `memcached-server:11211`.
```

**Review by oddlittlebird — APPROVED** [review]
Minor edits, then g2g.

**Review by kminehart — APPROVED** [review]

**Pull request merged**